### PR TITLE
Fix footer overflow

### LINKED ISSUE #31: [IMPROVEMENT] Footer needs to be made responsive on mobile devices
### Description

The Footer Component is currently not mobile responsive and it's causing a view overflow on the website

## Footer
WhatsApp Image 2023-10-21 at 13 02 43 (2)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Nathanael-R**
## Description
- Removed the experimental feature flag because the app router is now the default router, it's no longer an experimental feature.
 - Made the footer component responsive on mobile and fixed the screen overflow

## Related Issue
closes #31 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |

### Before

WhatsApp Image 2023-10-21 at 13 02 43 (2)

### After
WhatsApp Image 2023-10-25 at 23 19 47

## Testing Steps / QA Criteria

Clone the repo or pull the changes into your local environment and start it up using `npm run dev`, if you're sharing a network between devices, check out the localhost version on your mobile or just adjust/minimize your browser window as needed

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,8 +1,5 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-  experimental: {
```
Comment (YuriDevAT) [inline next.config.js:3]: What is the reason of deleting this part? Deleting this throws an error on my side 👇🏽 

Screenshot 2023-10-26 at 08 06 45

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -160,8 +160,8 @@ export default function Footer() {
               weekly.
             </p>
             <form className="sm:max-w-md mt-0">
-              <p className="text-sm text-gray-600 h-6">Enter you Email</p>
-              <div className="flex">
+              <p className="text-sm text-gray-600 h-6">Enter your Email</p>
```
Comment (YuriDevAT) [inline src/components/Footer.tsx:163]: Thanks for finding this typo!

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -160,8 +160,8 @@ export default function Footer() {
...
+              <div className="flex flex-col sm:flex-row gap-2 md:gap-0">
```
Comment (YuriDevAT) [inline src/components/Footer.tsx:164]: Nice implementation! 👏🏽

File: next.config.js (hunk 1 above)
Comment (Nathanael-R) [reply next.config.js:3]: That's odd, the screenshot below shows my terminal when I leave the experimental flag in the next config. It gives a psuedo warning. Adding the experimental flag doesn't not affect the code though, it only throws that warning. I'll fix it back in

Appdir

File: src/components/Footer.tsx (hunk 3 above)
Comment (Nathanael-R) [reply src/components/Footer.tsx:164]: Thank you.

**Review by Nathanael-R — COMMENTED** [review]
I've returned the experimental flag in the latest pr. It doesn't crash anything so there's no harm leaving it in there for now

**Pull request merged**